src/handlers/v1/markets/[chainId]/[marketId]/zap-supply/index.ts

```diff
@@ -163,7 +163,9 @@ export const v1GetZapSupplyQuotationRoute: Route<GetZapSupplyQuotationRouteParam
         );
       }
       const targetBorrowUSD = new BigNumberJS(borrowUSD);
-      const targetLiquidationThreshold = common.formatBigUnit(targetLiquidationLimit.div(targetCollateralUSD), 4);
+      const targetLiquidationThreshold = !targetLiquidationLimit.isZero()
+        ? common.formatBigUnit(targetLiquidationLimit.div(targetCollateralUSD), 4)
+        : '0';
```

Comment:
Do other operations have this issue as well?

Reply:
this issue is caused by `targetLiquidationLimit ==0` (no user collateral position).
considering all the scenarios we have:
- leverage     -> no issue due to user collateral position will > 0
- deleverage -> no issue due to user collateral position will > 0
- zap-borrow -> no issue due to user collateral position will > 0
- zap-repay   -> no issue due to user collateral position will > 0
- zap-supply     -> has this issue only when supplying base token (which is fixed in this PR)
- zap-withdraw -> has this issue when withdrawing base token or collateral (which is fixed in this PR)